Teams/04 Fundamentals of Microsoft Teams/Demos/01-teams-app-yeoman/solution/Readme.md

```diff
@@ -10,16 +10,27 @@ Head on over to [official documentation](https://msdn.microsoft.com/en-us/micros
 
 ## Building the app
 
+The application is built using the `build` Gulp task.
+
 ``` bash
 npm i -g gulp gulp-cli
 gulp build
 ```
 
 ## Building the manifest
 
+To create the Microsoft Teams Apps manifest, run the `manifest` Gulp task. This will generate and validate the package and finally create the package (a zip file) in the `package` folder. The manifest will be validated against the schema and dynamically populated with values from the `.env` file.
+
 ``` bash
 gulp manifest
 ```
+## Working with the devPreview manifest version
+
+You can change the manifest version from the standard `1.3` to `devPreview` by editing the `manifest.json` file and the `manifestVersion` property. The schema validation currently supports `1.3` and `devPreview`, for other options you need to manually edit the `gulpfile.js` file and add the version and schema in the `SCHEMAS` constant. When validating the manifest you will get an error if any other versions are used and you will also get a warning if you have not manually updated the schema reference in the `manifest.json` file (which is recommended for in-editor schema validation).
```

Comment:
I would say "You can change the manifest version to `devPreview`...", and remove the references to v1.3 here.

Reply:
This file is created by the Yeoman generator. 

We do not use preview releases for the Training Content, so perhaps this readme file gets deleted?

Reply:
Sorry, didn't notice the file path for this one. Go ahead and ignore.